runtime/drivers/mysql/parser.go

```diff
@@ -0,0 +1,224 @@
+package mysql
+
+import (
+	"database/sql"
+	"fmt"
+	"reflect"
+	"strings"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+)
+
+type mapper interface {
+	runtimeType(st reflect.Type) (*runtimev1.Type, error)
+	dest(st reflect.Type) (any, error)
+}
+
+// refer https://github.com/go-sql-driver/mysql/blob/master/fields.go for base types
+func getDBTypeNameToMapperMap() map[string]mapper {
+	m := make(map[string]mapper)
+
+	// bit
+	m["BIT"] = &bitMapper{}
+
+	// numeric
+	m["TINYINT"] = &numericMapper{}
+	m["SMALLINT"] = &numericMapper{}
+	m["MEDIUMINT"] = &numericMapper{}
+	m["INT"] = &numericMapper{}
+	m["UNSIGNED TINYINT"] = &numericMapper{}
+	m["UNSIGNED SMALLINT"] = &numericMapper{}
+	m["UNSIGNED INT"] = &numericMapper{}
+	m["UNSIGNED BIGINT"] = &numericMapper{}
+	m["BIGINT"] = &numericMapper{}
+	m["DOUBLE"] = &numericMapper{}
+	m["FLOAT"] = &numericMapper{}
+	// MySQL stores DECIMAL value in binary format
+	// It might be stored as string without losing precision
+	m["DECIMAL"] = &charMapper{}
+
+	// string
+	m["CHAR"] = &charMapper{}
+	m["LONGTEXT"] = &charMapper{}
+	m["MEDIUMTEXT"] = &charMapper{}
+	m["TEXT"] = &charMapper{}
+	m["TINYTEXT"] = &charMapper{}
+	m["VARCHAR"] = &charMapper{}
+
+	// binary
+	m[("BINARY")] = &byteMapper{}
+	m[("TINYBLOB")] = &byteMapper{}
+	m[("BLOB")] = &byteMapper{}
+	m[("LONGBLOB")] = &byteMapper{}
+	m[("MEDIUMBLOB")] = &byteMapper{}
+	m[("VARBINARY")] = &byteMapper{}
+
+	// date and time
+	m[("DATE")] = &dateMapper{}
+	m[("DATETIME")] = &dateMapper{}
+	m[("TIMESTAMP")] = &dateMapper{}
+	m[("YEAR")] = &int16Mapper{}
+	// TIME is scanned as bytes and can be converted to string
+	m[("TIME")] = &charMapper{}
+
+	// json
+	m[("JSON")] = &jsonMapper{}
```

Comment:
Why parenthesis around the map key strings?

Reply:
Removed the parenthesis